src/acquisition/covid_hosp/common/database.py

```diff
@@ -13,7 +13,11 @@
 
 class Database:
 
-  def __init__(self, connection, table_name=None, columns_and_types=None):
+  def __init__(self,
+               connection,
+               table_name=None,
+               columns_and_types=None,
+               additional_fields=tuple()):
     """Create a new Database object.
 
     Parameters
```

Comment:
do you mind updating the docstring below with the new `additional_fields` param?

Reply:
updated